cds-snc/notification-admin
# New feature flag `FF_SMS_PARTS_UI`

# Summary | Résumé

Part of https://github.com/cds-snc/notification-planning/issues/901

I created the new feature flag `FF_SMS_PARTS_UI` and put UI components behind it that are not part of the V1 release of the SMS Limits feature.

I also added a test to check that the relevant section is shown/hidden on the "review before sending" page depending on the feature flag.

## To test

Check out branch, set the value of the new feature flag to `True` and `False`, checking that the relevant sections are visible.
